components/atom/card/src/index.scss

```diff
@@ -0,0 +1,34 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$c-atom-card-highlight: #fffed4 !default;
+
+.sui-AtomCard {
+  display: flex;
+
+  &-media {
+    width: 33%;
+  }
+
+  &-info {
+    padding: 16px;
+  }
+
+  &-link {
+    color: inherit;
+    text-decoration: none;
+  }
+
+  &-highlight {
```

Comment:
should this really be part of this component? IMO the content and media components should be responsible of their behaviours on highlight

Reply:
Not sure about this as this an specification of the `AtomCard` and the `media` and `content` components could be anything so I don't see how to delegate this responsibility on them and a strong reason for doing that

Reply:
Both highlight and vertical seems to be a modifier and not an element so you should use two `--` for the classname.